src/buffer/out/search.cpp

```diff
@@ -97,11 +97,7 @@ bool Search::FindNext()
 // - Takes the found word and selects it in the screen buffer
 void Search::Select() const
 {
-    // Only select if we've found something.
-    if (_coordSelStart != _coordSelEnd)
-    {
-        _uiaData.SelectNewRegion(_coordSelStart, _coordSelEnd);
-    }
+    _uiaData.SelectNewRegion(_coordSelStart, _coordSelEnd);

```

Comment:
Why was the conditional surrounding this line removed?

Reply:
With the surrounding condition, the search module cannot support single letter search. However, because that FindNext() only returns true if there is a match, as long as we check before we select, there should be a text match (including a single letter. )

Reply:
Ah okay. It's a [`_coordSelStart`, `_coordSelEnd`] selection, not a [`_coordSelStart`, `_coordSelEnd`) - that makes more sense then.